packages/cli/src/commands/create/index.ts

```diff
@@ -131,21 +142,55 @@ async function _createTutorial(flags: CreateOptions) {
 
   updatePackageJson(resolvedDest, tutorialName, flags);
 
-  const { selectedPackageManager, dependenciesInstalled } = await installDependencies(resolvedDest, flags);
+  const selectedPackageManager = await selectPackageManager(flags);
 
   updateReadme(resolvedDest, selectedPackageManager, flags);
 
   await setupEnterpriseConfig(resolvedDest, flags);
 
   await initGitRepo(resolvedDest, flags);
 
+  const { install, start } = await installAndStart(flags);
+
   prompts.log.success(chalk.green('Tutorial successfully created!'));
 
-  printNextSteps(dest, selectedPackageManager, dependenciesInstalled);
+  if (install || start) {
+    let message = 'Please wait while we install the dependencies and start your project...';
 
-  prompts.outro(`You're all set!`);
+    if (install && !start) {
+      // change the message if we're only installing dependencies
+      message = 'Please wait while we install the dependencies...';
 
-  console.log('Until next time 👋');
+      // print the next steps without the install step in case we only install dependencies
+      printNextSteps(dest, selectedPackageManager, true);
+    }
+
+    prompts.outro(message);
+
+    await startProject(resolvedDest, selectedPackageManager, flags, start);
+  } else {
+    printNextSteps(dest, selectedPackageManager, false);
+
+    prompts.outro(`You're all set!`);
+
+    console.log('Until next time 👋');
+  }
+}
+
+async function startProject(cwd: string, packageManager: PackageManager, flags: CreateOptions, startProject: boolean) {
+  if (flags.dryRun) {
+    const message = startProject
+      ? 'Skipped dependency installation and project start'
+      : 'Skipped dependency installation';
+
+    console.warn(`${warnLabel('DRY RUN')} ${message}`);
```

Comment:
We don't do anything in case of a dry run.

```
◆  Tutorial successfully created!
│
└  Please wait while we install the dependencies and start your project...

 DRY RUN  Skipped dependency installation and project start
```